File: python-script/weather-dummy.py
import random

def get_weather(city):
    weathers = ["Cerah", "Mendung", "Hujan", "Gerimis", "Badai"]
    temp = random.randint(20, 35)
    print(f"Cuaca di {city}: {random.choice(weathers)}, {temp}°C")

if __name__ == "__main__":
    city = input("Masukkan nama kota: ")
    get_weather(city)

File: java-script/WeatherDummy.java
// WeatherDummy.java
import java.util.Random;
import java.util.Scanner;

public class WeatherDummy {
    public static void main(String[] args) {
        String[] kondisi = {"Cerah", "Mendung", "Hujan", "Gerimis", "Badai"};
        Scanner input = new Scanner(System.in);
        System.out.print("Masukkan nama kota: ");
        String kota = input.nextLine();

        Random rand = new Random();
        int suhu = 20 + rand.nextInt(16);
        String cuaca = kondisi[rand.nextInt(kondisi.length)];

        System.out.println("Cuaca di " + kota + ": " + cuaca + ", " + suhu + "°C");
        input.close();
    }
}
